tests/phpunit/tests/rest-api/rest-block-navigation-areas-controller.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Unit tests covering WP_REST_Block_Navigation_Areas_Controller functionality.
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 5.9.0
+ */
+class WP_Test_REST_Block_Navigation_Areas_Controller extends WP_Test_REST_Controller_Testcase {
+
+	const OPTION = 'fse_navigation_areas';
+
+	/**
+	 * @var int Administrator user ID
+	 */
+	protected static $admin_id;
+
+	/**
+	 * @var array Contains mapping
+	 */
+	protected static $old_mapping;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$admin_id = $factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+
+		static::$old_mapping = get_option( static::OPTION, array() );
+	}
+
+	public static function wpTearDownAfterClass() {
+		self::delete_user( self::$admin_id );
+		update_option( static::OPTION, static::$old_mapping );
+	}
+
+	public function test_get_items() {
+		wp_set_current_user( static::$admin_id );
+		$request = new WP_REST_Request( Requests::GET, '/wp/v2/block-navigation-areas' );
```

Comment:
```suggestion
		$request = new WP_REST_Request( 'GET', '/wp/v2/block-navigation-areas' );
```

I'd just type out the request methods here.